src/openMVG/matching/regions_matcher.hpp

```diff
@@ -222,14 +223,45 @@ class RegionsMatcherT : public RegionsMatcher
       vec_fDistance.begin(), // distance start
       vec_fDistance.end(),   // distance end
       NNN__, // Number of neighbor in iterator sequence (minimum required 2)
-      vec_nn_ratio_idx, // output (indices that respect the distance Ratio)     
+      vec_valid_matches_idx, // output (indices that respect the distance Ratio)     
       b_squared_metric_ ? Square(f_dist_ratio) : f_dist_ratio);
 
-    vec_putative_matches.reserve(vec_nn_ratio_idx.size());
-    for (size_t k=0; k < vec_nn_ratio_idx.size(); ++k)
+    const size_t n = vec_fDistance.size();
+    
+    // Consider the match valid if the 2 first matches correspond to the "same" feature.
+    //
+    // In [Mishkin 2015] "MODS: Fast and Robust Method for Tow-View Matching" they introduce the FGINN strategy
```

Comment:
Thank you @fabiencastan for the explanation:
Just a paper sharing regarding the context:
http://web.stanford.edu/~bgirod/pdfs/ChenVCIP09.pdf